### PR TITLE
feat: allow telegram bot send to group

### PR DESCRIPTION
**Pull request by jnlin**
This patch allows telegram bot to send message to a telegram group, to aggregate all notification in 1 place

### CODE REVIEW

File: pkg/notifier/telegramnotifier/interaction.go [hunk 1]
```diff
@@ -60,14 +62,14 @@ func (it *Interaction) Session() *Session {
 }
 
 func (it *Interaction) HandleInfo(m *telebot.Message) {
-	if it.session.Owner == nil {
+	if it.session.Owner == nil || it.session.Chat == nil {
 		return
 	}
 
 	if m.Sender.ID != it.session.Owner.ID {
```
Comment (c9s) [inline pkg/notifier/telegramnotifier/interaction.go:67]: you need to check Owner == nil here i guess?

**Review by c9s — APPROVED** [review]
awesome!

File: pkg/notifier/telegramnotifier/interaction.go (hunk 1 above)
Comment (jnlin) [reply pkg/notifier/telegramnotifier/interaction.go:67]: Yes. If (Owner == nil OR Chat == nil), then stop handling request

Comment (c9s) [reply pkg/notifier/telegramnotifier/interaction.go:67]: got it, thanks~

**Pull request merged**